update

DimaHavr opened the pull request:
Додано сторінку з послугами для гаджета;
Додано анімацію на модалки;
Додано ховери;
Змінено фон футера;
Змінено назву папки компонентів;

DimaHavr committed:
update

Додано сторінку з послугами для гаджета;
Додано анімацію на модалки;
Додано ховери;
Змінено фон футера;
змінено назву папки компонентів;

DimaHavr committed:
fix body overflow

DimaHavr committed:
Update SubcategorySection.tsx

DimaHavr committed:
update

DimaHavr committed:
fix logo

Iryna-Gry commented on app/(layouts)/(components)/CourierModal.tsx:
Зроби анімацію 0.1 на бекдропі на 0.3 на модалці усюди.

Iryna-Gry commented on app/(pages)/repair/(components)/CategorySection.tsx:
прибери 425 обмеження бо виходить отаке, на таблеті можна зробити саме таким чином
![image](https://github.com/Iryna-Gry/fix-lab-project/assets/108401183/b5baa76a-25d5-4d15-8842-5d4bd63f2bc1)

Iryna-Gry commented on app/(pages)/repair/(components)/GadgetServicesList.tsx:
теж саме що і попереднє

Iryna-Gry commented on app/layout.tsx:
що таке bodyPadding? не знайшла

DimaHavr committed:
update

Пофіксив зауваження з ревью;
Виправив здвиги сторінки підчас відкривання модалки, меню
